x/bank/keeper/genesis.go

```diff
@@ -42,17 +43,24 @@ func (k BaseKeeper) ExportGenesis(ctx sdk.Context) types.GenesisState {
 		return false
 	})
 
+	addrs := make([]string, 0, len(balancesSet))
+	for a := range balancesSet {
```

Comment:
can we just replace the `balancesSet` for  `[]Balance`? Ideally wrap the iterator in `k.GetAllBalances(ctx)`